pyomo/mpec/plugins/solver2.py

```diff
@@ -39,7 +39,7 @@ def _apply_solver(self):
         xfrm.apply_to(self._instance)
 
         xfrm = TransformationFactory('gdp.bigm')
-        xfrm.apply_to(self._instance, default_bigM=self.options.get('bigM',10**6))
+        xfrm.apply_to(self._instance, bigM=self.options.get('bigM',10**6))
```

Comment:
Totally minor, but why 10**6 vs 100000 used in an earlier file as the default? (1) I think it was 1e6 vs 1e5, but I am not sure. Also, what is wrong with 1e5? or 1e6?

Reply:
I believe that @whart222 wrote this code and that was his style.  I agree that 1e6 would be better.